src/main/webapp/i18n/en/guidedTour.json

```diff
@@ -13,7 +13,7 @@
         },
         "cancel": {
             "headline": "Tutorial cancelled",
-            "content": "You have cancelled the tutorial. If you want to continue this tutorial at any other time, then click on <tt>Continue tutorial</tt> from the account menu."
+            "content": "You have cancelled the tutorial. If you want to continue this tutorial at another time, click on <tt>Continue tutorial</tt> from the account menu."
```

Comment:
Ahah! This is a matter of British English vs American English. I didn't think to change "cancelled" because that is the form I am used to as an American. The Artemis team may have a policy of using BRE, and that's fine. I noticed that "modeling" is also spelled with only one L, which is distinctly British.

Reply:
Hold on, just Googled this. Apparently "cancelled" is British English after all and I've been using that version all my life without knowing it. In that case I would suggest leaving it as-is.

Reply:
According to Grammarly I am dead wrong about spellings with one L being British. It comes as a big surprise to me, but that's purely an American thing! I am not used to those spellings at all. I think I will make another commit changing "modeling" to "modelling"; that is the more international spelling, after all.

Reply:
in any case, we would like to consistently use American English and **`not`** British English